src/components/Pagination/Pager.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { noop } from '../../common/helpers';
+
+/**
+ * Pager component for Patternfly React
+ */
+const Pager = ({
+  baseClassName,
+  className,
+  messages,
+  disableNext,
+  onNextPage,
+  disablePrevious,
+  onPreviousPage
+}) => {
+  const classes = cx('pager', className);
+  return (
+    <ul className={classes}>
+      <li className={cx({ disabled: disablePrevious }, 'previous')}>
+        <a
+          href="#"
+          className={cx({ disabled: disablePrevious })}
+          onClick={e => {
+            e.preventDefault();
+            if (!disablePrevious) {
+              onPreviousPage(e);
+            }
+          }}
+        >
+          <span className="i fa fa-angle-left" />
```

Comment:
oooo forgot one other thing... can we use `<Icon name="angle-left">` or something like that?